test/1.4/serialize/T_saved_dead_vtable.dml

```diff
@@ -0,0 +1,29 @@
+/*
+  © 2022 Intel Corporation
+  SPDX-License-Identifier: MPL-2.0
+*/
+dml 1.4;
+
+device test;
+
+/// COMPILE-ONLY
+
+template t {
+    shared method m();
+}
+
+template u {
+    shared method n();
+}
+
+group g is u {
+    method n() {}
```

Comment:
Maybe add a negative test case, like
```
trait x { shared method m(); }
trait y { shared method n(); }
/// WERRSTMT
group x is x { method m() { error; }}
// no WERRSTMT because unused (weird)
group y is y { method n() { error; }}
saved x z;
```

Reply:
That's already checked in `T_trait_dce`.

Reply:
Actually, no, it isn't. I'd rather update `T_trait_dce`.